install/kubernetes/webhook-create-signed-cert.sh

```diff
@@ -115,14 +109,9 @@ done
 kubectl certificate approve ${csrName}
```

Comment:
As a note: another way is to pull the istio key pair ( created automatically by istio-ca), and get it signed by kubernetes. You end up with 2 certs - one working for istio, one for k8s - and avoid dep on cfssl. 
As another note, the testing in CI are using another tool to generate certs - we should eventually pick one and have 
it installed in the CI builder.

Reply:
`webhook-create-signed-cert.sh` is not a hard dependency. It simply demonstrates how to use the k8s CA and existing OSS tools to generate a signed cert. Other options exist including istio-ca or user's existing PKI. The underlying webhook shouldn't care.

Reply:
Does istio-ca generate cert with CN? @wattli 

re: testing, the current e2e pilot test uses a self-signed cert. How important is it that those tests align with other test cert generation?

Reply:
I meant to use istio-ca key pair - the cert still needs to be signed/generated by k8s.